test-resources/otlp-sink/otlp-sink.yaml

```diff
@@ -60,7 +65,7 @@ metadata:
   labels:
     app.kubernetes.io/name: deployment
     app.kubernetes.io/component: manager
-    dash0.com/enable: "false"
+    dash0.com/instrumented: "false"
```

Comment:
@mmanciop last review/looking over the changes before merging. I don't understand this particular change. 

It probably does not matter, but `dash0.com/instrumented` is typically something that the operator would set. I would not expect that it is set externally. I think the previous choice `dash0.com/enable: "false"` made a little more sense. In the end, neither label should have an effect and we could probably delete it altogether, since there is never a Dash0Monitoring resource in the otlp-sink namespace anyway.